app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -315,4 +317,8 @@ public static String formLoadPayloadStatus() {
     public static boolean isDetailTabSwipeActionEnabled() {
         return doesPropertyMatch(DETAIL_TAB_SWIPE_ACTION_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
     }
+
+    public static boolean isOfflineUpdateEnabled() {
+        return doesPropertyMatch(OFFLINE_UPDATE_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
```

Comment:
My thinking was that projects may want to disable it via the custom flag thing on HQ. Does that sound improbable?